src/parameterized_tendencies/microphysics/microphysics_wrappers.jl

```diff
@@ -160,11 +160,13 @@ function compute_precipitation_sources!(
     mp,
     thp,
 )
-    FT = eltype(Sqₜᵖ)
-    @. Sqₜᵖ = FT(0)
-    @. Sqᵣᵖ = FT(0)
-    @. Sqₛᵖ = FT(0)
-    @. Seₜᵖ = FT(0)
+    FT = eltype(thp)
+    # @. Sqₜᵖ = FT(0) should work after fixing
+    # https://github.com/CliMA/ClimaCore.jl/issues/1786
```

Comment:
Thanks @trontrytel. Can you please update the news file as well to mark this change?